webapp/src/selectors.js

```diff
@@ -17,3 +17,32 @@ export const getMessage = (state) => {
     return post.message;
 };
 export const getItems = (state) => getPluginState(state).items;
+export const getCurrentTeamRoute = (state) => {
+    const basePath = getSiteURL();
+    const teamName = state.entities.teams.teams[state.entities.teams.currentTeamId].name;
+
+    return basePath + '/' + teamName + '/';
+};
+
+function getSiteURL() {
+    let siteURL = '';
```

Comment:
1/5 suggestion to set the value first and remove the `else`
```suggestion
    let siteURL = siteURL = window.location.protocol + '//' + window.location.hostname + (window.location.port ? ':' + window.location.port : '');
```
or `siteURL = window.location.origin` and use `if (!window.location.origin)`